### PR TITLE
feat(color-property): Add color property support

### PR DESCRIPTION
**Pull request by zeucxb**
> Add support to color properties

ezgif com-video-to-gif

### CODE REVIEW

File: lib/property_widgets.dart [hunk 1]
```diff
@@ -131,18 +140,94 @@ class ListPropertyState extends State<ListPropertyWidget> {
...
+  State<StatefulWidget> createState() =>
+      ColorPropertyState(property.getValue());
+}
+
+class ColorPropertyState extends State<ColorProperty> {
+  // create some values
+  Color pickerColor;
+  Color currentColor;
+
+// ValueChanged<Color> callback
```
Comment (erickzanardo) [inline lib/property_widgets.dart:183]: This comment should aligned with the method indentation

File: lib/property_widgets.dart [hunk 2]
```diff
@@ -131,18 +140,94 @@ class ListPropertyState extends State<ListPropertyWidget> {
...
+  void changeColor(Color color) {
+    setState(() => pickerColor = color);
+  }
+
+// raise the [showDialog] widget
```
Comment (erickzanardo) [inline lib/property_widgets.dart:188]: This comment should aligned with the method indentation

File: lib/property_widgets.dart [hunk 3]
```diff
@@ -131,18 +140,94 @@ class ListPropertyState extends State<ListPropertyWidget> {
...
+
+  @override
+  Widget build(BuildContext context) {
+    return _PropertyScaffold(
+      label: widget.property.name,
+      child: RaisedButton(
+        elevation: 0,
+        color: currentColor,
+        onPressed: () async {
+          await show();
```
Comment (erickzanardo) [inline lib/property_widgets.dart:226]: Is show async? It has not return type

File: lib/widget.dart [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import 'package:flutter/material.dart';
 import 'package:flutter/foundation.dart';
 import './story.dart';
-import './property_widgets.dart';
+import './property_widgets.dart' as p;
```
Comment (erickzanardo) [inline lib/widget.dart:4]: Any reason why you needed to make that alias?

**Review by erickzanardo — COMMENTED** [review]
Left some comments, but this looks awesome, thanks a lot for this contribution.

Just please make sure that this property works fine on web.

File: lib/widget.dart (hunk 4 above)
Comment (zeucxb) [reply lib/widget.dart:4]: ColorProperty already exists 😑

Comment (erickzanardo) [reply lib/widget.dart:4]: Probably on `material.dart` right? Alright, fair enough!

**Review by erickzanardo — APPROVED** [review]
LGTM, I shall release this soon

**Pull request merged**